query-coordinator/src/test/scala/com/socrata/querycoordinator/TestQueryRewriter.scala

```diff
@@ -16,26 +22,35 @@ class TestQueryRewriter extends TestQueryRewriterBase {
     * i.e. non-system columns prefixed by an _, and backtick escaped
```

Comment:
this doesn't seem accurate anymore; I also don't follow why its being changed?

Reply:
Updated